omero/sysadmins/unix/install-web-trial.txt

```diff
@@ -0,0 +1,348 @@
+OMERO.web WSGI deployment (TRIAL)
+=================================
+
+OMERO.web is the web application component of the OMERO platform which
+allows for the management, visualization (in a fully multi-dimensional
+image viewer) and annotation of images from a web browser. It also
+includes the ability to manage users and groups.
+
+Deploying OMERO.web separately to OMERO.server is recommended as they
+perform best under different circumstances and require a different set of
+dependencies. This guide uses the example of deploying with nginx and
+gunicorn.
+
+Prerequisites
+-------------
+
+-  Python 2.7
+
+   -  :pypi:`virtualenv` (optional) tool to create isolated Python environments
+
+   -  :pypi:`PyPI <>` a package management system used to install and manage
+      software packages written in Python. PyPI is already installed if
+      you are using Python 2 >=2.7.9
+
+   -  `omego <https://github.com/ome/omego>`_ OME package management system
+
+   -  `ZeroC`_ IcePy 3.6 (IcePy 3.5 is also supported although requires global
+      installation from RPM)
```

Comment:
> How sysadmin prefer to install. 

is an interesting question that certainly need not block this PR. For instance, I have Pillow <3 because I installed Debian 8's `python-pil` (as we use newer features in our playbooks, things like ansible and shade I think I pulled in from Debian's packages for what will be version 9). Making much use of multiple package systems on the one machine just leads to confusion -- in knowing even which system to ask why this file is here, keeping up to date with security fixes, sweeping away cruft, etc. -- fortunately OMERO requires popular enough stuff that I can largely keep everything managed by the one system. Maybe I'm unusual, but I much prefer to use my distribution's package manager rather than that *and* an extra package manager for every language I need extra stuff for: pip, cpan, hackage, etc. It also makes it simpler to restore state to a fresh system. However, @rleigh-codelibre or @kennethgillen for instance may have a wholly different view.